lib/cubit/account/account_state.dart

```diff
@@ -71,7 +78,7 @@ extension WalletInfoToJson on WalletInfo {
 extension WalletInfoFromJson on WalletInfo {
   static WalletInfo? fromJson(Map<String, dynamic>? json) {
     if (json == null) {
-      _logger.info('walletInfo is missing from AccountState JSON.');
+      _logger.info('walletInfo is missing from WalletInfo JSON.');
```

Comment:
This naming is actually correct as the parent JSON is of `AccountState`'s. 

These changes need to be reverted.

```suggestion
      _logger.info('walletInfo is missing from AccountState JSON.');
```